cellxgene_schema_cli/cellxgene_schema/validate.py

```diff
@@ -182,6 +182,39 @@ def _validate_curie_ontology(self, term_id: str, column_name: str, allowed_ontol
                 f"'{term_id}' in '{column_name}' is not a valid ontology term id of '{', '.join(allowed_ontologies)}'."
             )
 
+    def _validate_curie_str(self, term_str: str, column_name, curie_constraints: dict):
+        """
+        Validate a curie str based on some constraints. If there are any errors, it adds them to self.errors
+
+        :param str term_str: the curie term str to validate
+        :param str column_name: Name of the column in the dataframe
+        :param dict curie_constraints: constraints for the curie term to be validated,
+        this part of the schema definition
+
+        :rtype None
+        """
+        # If there are exceptions and this is one then skip to end
+        if "exceptions" in curie_constraints and term_str in curie_constraints["exceptions"]:
+            return
+
+        # If NA is found in allowed ontologies, it means only exceptions should be found. If no exceptions were found
+        # then return error
+        if curie_constraints["ontologies"] == ["NA"]:
+            self.errors.append(f"'{term_str}' in '{column_name}' is not a valid value of '{column_name}'.")
+            return
+
+        # if multi_term is defined, split str into individual ontology terms and validate each
+        if curie_constraints.get("multi_term", None):
+            delimiter = curie_constraints["multi_term"]["term_delimiter"]
+            term_ids = term_str.split(delimiter)
+            for term_id in term_ids:
+                self._validate_curie(term_id, column_name, curie_constraints)
+            if curie_constraints["multi_term"].get("sorted", False) and len(term_ids) > 1:
```

Comment:
```suggestion
            if "sorted" in curie_constraints["multi_term"] and len(term_ids) > 1:
```

Reply:
I think if we're going to introduce a boolean field, we should consider that someone might try to assign `sorted: False` in the future and account for that in validation